run.go

```diff
@@ -97,18 +97,39 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 	if err != nil {
 		return err
 	}
+	boundMounts := []specs.Mount{}
 	defer func() {
 		if err2 := b.Unmount(); err2 != nil {
 			logrus.Errorf("error unmounting container: %v", err2)
 		}
 	}()
 	g.SetRootPath(mountPoint)
 	g.SetProcessTerminal(true)
+	g.RemoveLinuxNamespace("network")
 	spec := g.Spec()
 	if spec.Process.Cwd == "" {
 		spec.Process.Cwd = DefaultWorkingDir
 	}
 	mounts := options.Mounts
+	for _, boundFile := range []string{"/etc/hosts", "/etc/resolv.conf"} {
+		override := false
+		for _, mount := range mounts {
+			if mount.Destination == boundFile {
+				// Already have an override for it, so skip this one.
+				continue
+			}
+		}
+		if override {
+			continue
+		}
+		boundMount := specs.Mount{
+			Source:      boundFile,
+			Destination: boundFile,
+			Type:        "bind",
```

Comment:
Source is the path on the host that we want to bind into the container.  In this case we're defaulting to using read-only bind mounts of the host's `/etc/hosts` and `/etc/resolv.conf`, refraining from doing so for either of them if any of the `Mount` values passed in via `options` also specifies the same destination.